packages/cli/src/util/dev/builder.ts

```diff
@@ -288,7 +288,7 @@ export async function executeBuild(
   // Convert the JSON-ified output map back into their corresponding `File`
   // subclass type instances.
   for (const name of Object.keys(output)) {
-    const obj = output[name] as File;
+    const obj = output[name] as File | Lambda;
```

Comment:
This is a bit of a hammer to make the types work. This chunk of code should probably be refactored later to use the newer `File` types. (That's what @TooTallNate tells me, anyway!)

Reply:
What about the `Files` type. I think thats like a list of `File` objects but it is sometimes uses as the "Output" (which would include `Lambda`).